backend/src/packages/business/business.service.ts

```diff
@@ -20,12 +20,12 @@ class BusinessService implements IService {
     this.businessRepository = businessRepository;
   }
 
-  public async find(
+  public async findById(
     id: number,
-  ): Promise<OperationResult<BusinessEntityT | null>> {
-    const business = await this.businessRepository.find(id);
+  ): Promise<BusinessEntityT | null> {
+    const result = await this.businessRepository.find({ id });
 
-    return { result: business ? business.toObject() : null };
+    return result.length === 1 ? BusinessEntity.initialize(result[0]).toObject() : null;
```

Comment:
const [business = null] = await 

return business ? BusinessEntity.initialize(business).toObject() : null;

Reply:
nice. I wondered, why is it not working. It is because of default null.